### ISSUE TITLE
A bug while <> symbol interpretation

### ISSUE THREAD

**Issue by hipeta** [hipeta/arrow-macros#1]
Since using flatten in diamond wand,  this test below is not passed.

```
(test <>-symbol-interpretation-test
  (is (equal (-<>> (list 1 2 3 4 5)
               (mapcar (lambda (x) (-<> x (+ <> <>)))))
             (-<>> (list 1 2 3 4 5)
               (mapcar (lambda (x) (-<> x (+ <> <>))) <>)))))
```

Only using code walker solves this.

**Comment by hipeta** [comment]
Fixed.

**Comment by hipeta** [comment]
This bug was fixed on commit b5565ce7be by using hu.dwim.walker on linux, but I cannot build hu.dwim.walker on windows, so I revert this commit and reopen this issue.

**Comment by hipeta** [comment]
hu.dwim.walker is now able to be built on windows too (thanks to Attila Lendvai!), so I revert the commit which reverted b5565ce and close this issue.
